wizard_agent/brain/treasure_strategy.py

```diff
@@ -20,20 +20,47 @@ def _get_nearest_treasure(location, treasure_list):
     else:
         return None
 
+def _get_furthest_treasure_from_opponent(opponent_location, treasure_list):
+    if treasure_list and len(treasure_list) > 1:
+        treasure_distance = 10
```

Comment:
should start at 0 because furthest could be less than 10

Reply:
ok